templates/bower-json.js

```diff
@@ -1,15 +1,18 @@
 module.exports = (answers) => {
 	return `{
-  "name": "${answers.name}",
-  "description": "${answers.description}",
-  "ignore": [
-    "node_modules",
-    "bower_components",
-    "build"
-  ],
-  "main": [
-    ${answers.scss ? `"main.scss"` : ''}${answers.scss && answers.javascript ? "," : ''}
-    ${answers.javascript ? `"main.js"` : ''}
-  ]
+	"name": "${answers.name}",
+	"description": "${answers.description}",
+	"ignore": [
+		"node_modules",
+		"bower_components",
+		"build"
+	],
+	"main": [
+		${answers.scss ? `"main.scss"` : ''}${answers.scss && answers.javascript ? "," : ''}
+		${answers.javascript ? `"main.js"` : ''}
+	],
+	"dependencies": {
+		${answers.brand ? `"o-brand": "*"` : ''}
```

Comment:
Some brand functions are use in the boilerplate, so maybe we should put a version here?

Reply:
```suggestion
		${answers.brand ? `"o-brand": "^3.2.10"` : ''}
```